skimage/feature/corner_cy.pyx

```diff
@@ -78,8 +78,8 @@ def _corner_moravec(image, Py_ssize_t window_size=1):
                             msum = 0
                             for mr in range(- window_size, window_size + 1):
                                 for mc in range(- window_size, window_size + 1):
-                                    msum += (cimage[r + mr, c + mc]
-                                             - cimage[br + mr, bc + mc]) ** 2
+                                    t = cimage[r + mr, c + mc] - cimage[br + mr, bc + mc]
```

Comment:
Cython compilation is failing here, because `cimage` is a `double` while new variable `t` is `Py_ssize_t` in this file.